src/usb/usb-request.h

```diff
@@ -22,7 +22,7 @@ namespace librealsense
         {
         public:
             virtual rs_usb_endpoint get_endpoint() const = 0;
-            virtual int get_actual_length() const = 0;
+            virtual uint64_t get_actual_length() const = 0;
```

Comment:
Dangerous -- and need to keep the type for "length" uniform -- so either `int`, `size_t`, or `uint64_t`
I prefer to keep `int` and not change interfaces for this. Please see if you can change the usages rather than the definition.

Reply:
done